src/charselect.cpp

```diff
@@ -305,7 +306,7 @@ void Courtroom::character_loading_finished()
     }
   }
   ui_char_list->expandAll();
-  filter_character_list();
+//  filter_character_list();
```

Comment:
?

Reply:
Caused a segfault when forcing a client into CSS after updating the char list